MichaelAng/dnd-char-gen
# Feature/xx/webpack restructure

# Issue

PR #10 
PR #23 
PR #24 
# Changes
- Add bootstrap fonts
- Add bootstrap css
- Create Production Environment
- Add Client unit testing
- Add Linting
# Steps to test
1. Manually Test using the following steps
   1. Run `npm install`
   2. Run `npm start`
   - [x] Verify the application starts
   1. Run `npm test`
   - [x] Verify test works
   1. Run `npm run build && cd dist`
   2. Run a webserver for example `http-server`
   - [x] Verify the application starts with the dist file
   1. Inspect the browser 
   - [x] Verify you get a bundled application that is unminified 
2. Check if the system build with a production environment 
   1. export NODE_ENV=production
   2. Run `npm install`
   3. Run `npm start`
   - [x] Verify the application starts
   1. Run `npm test`
   - [x] Verify test works
   1. Run `npm run build && cd dist`
   - [x] Verify the application starts with the dist file
   1. Inspect the browser  
   - [x] Verify you get a bundled application that is minified
# Help

[wiki](https://github.com/MichaelAng/dnd-char-gen/wiki)
